server/src/main/java/org/solarbank/server/dto/EnergyTariff.java

```diff
@@ -0,0 +1,17 @@
+package org.solarbank.server.dto;
+
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class EnergyTariff {
+    @NotBlank
+    private String currencyCode;
```

Comment:
We should probably validate this is a valid currency code using the java money library:

https://www.baeldung.com/java-money-and-currency

Reply:
Added validation for this. Created an exception handler for UnknownCurrency, but gets overridden by the default exception handler. Not sure how to fix this